frontend/app/components/floor-panel/component.js

```diff
@@ -0,0 +1,47 @@
+import Ember from 'ember';
+
+const get = Ember.get;
+const set = Ember.set;
+
+export default Ember.Component.extend({
+  store: Ember.inject.service(),
+
+  isAddingBeer: false,
+
+  init() {
+    this._super(...arguments);
+
+    this.searchResults = Ember.A();
```

Comment:
`this.set('searchResults', Ember.A())` ??

can it be `set(this.searchResults, Ember.A())` or `set(this, 'searchResults', Ember.A())`